src/backend/kms/surface/mod.rs

```diff
@@ -683,7 +689,8 @@ impl SurfaceThreadState {
             }
         };
 
-        let _ = tx.send(());
+        let _ = tx.send(Ok(()));
```

Comment:
It seems to me like this can only ever return `Ok(())`. Given we don't handle the result in `pause_session` (which is bad practice, should this ever return an error), I would rather keep the sender/receive, etc sending a `()`, like it did before.

So the only thing that needs to change in this commit imo, is the reordering for `device.drm.pause()`. The rest can be dropped.